.internal-ci/README.md

```diff
@@ -0,0 +1,191 @@
+# Mobilecoin Internal CI/CD tools
+
+These build, dockerfiles, helm charts and scripts are used internally for MobileCoin builds and deployments and is subject to change without notice, YMMV, no warranties provided, all that good stuff.
+
+## Workflow
+
+- Members of the `mobilecoinfoundation` org should use the Branching Workflow.
```

Comment:
I strongly recommend avoiding SDLC proscriptions in this PR, unless you'd like to have that discussion here.

Reply:
Removed the workflow section.  It would be really nice to have that defined so people know how to interact with the system.